addons/project/wizard/mail_invite_share_inherit.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, models, fields
+
+
+class MailInviteShare(models.TransientModel):
+    _inherit = 'mail.invite.share'
+
+    @api.model
+    def default_get(self, fields):
```

Comment:
Well if I understand well there are some documents you could try to share (here taks) that actually have no portal access. It should be implemented in the mixin with overrides where necessary. That's not the role of a wizard to compute the accessibility of the portal. The wizard should only find the URL and send the email. It should not be business-decisions dependent.